frontend/src/features/rollout-announcement/components/LastFeatureContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, Container, ModalBody, ModalHeader, Text } from '@chakra-ui/react'
+
+interface FeatureUpdate {
+  title: string
+  ImageSvgr: JSX.Element
+}
+
+interface LastFeatureContentProps {
+  updates: FeatureUpdate[]
+}
+
+export const LastFeatureContent = (
+  props: LastFeatureContentProps,
+): JSX.Element => {
+  const { updates } = props
```

Comment:
can destructure in the function arguments directly

Reply:
Okay, updated PR to destructure in the function args